backend/src/app/auth.gleam

```diff
@@ -0,0 +1,34 @@
+import birl
+import gleam/json
+import gleam/result
+import gwt
+import simplifile
+import wisp
+
+pub fn generate_jwt(userid: Int, username: String) {
+  let builder =
+    gwt.new()
+    |> gwt.set_issuer("hardwareby.us")
+    |> gwt.set_subject(username)
+    |> gwt.set_audience("hardwareby.us")
+    |> gwt.set_issued_at(birl.to_unix(birl.now()))
+    |> gwt.set_expiration(birl.to_unix(birl.now()) + 600)
+    |> gwt.set_payload_claim("userid", json.int(userid))
```

Comment:
I'm not super familiar with JWTs, is this needed for us to track the user associated with this token?